### PR TITLE
[homebridge] New chart

### PR DESCRIPTION
**Pull request by bjw-s**
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[radarr]`)

### CODE REVIEW

File: charts/homebridge/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+appVersion: 3.1.0
+version: 1.0.0
+name: homebridge
+description: A lightweight NodeJS server that emulates the iOS HomeKit API
+type: application
+keywords:
+  - homebridge
+  - homekit
+home: https://homebridge.io/
```
Comment (billimek) [inline charts/homebridge/Chart.yaml:10]: For `home:` Can you please use `https://github.com/k8s-at-home/charts/tree/master/charts/homebridge` and put `https://homebridge.io/` under `sources:`?

File: charts/homebridge/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+appVersion: 3.1.0
+version: 1.0.0
+name: homebridge
```
Comment (billimek) [inline charts/homebridge/Chart.yaml:4]: It's a minor thing, but can you include an icon for this chart as well?  It makes it looks a lot nicer when browsing a place like [artifacthub](https://artifacthub.io/packages/search?page=1&org=k8s-at-home)

**Review by billimek — APPROVED** [review]
Thanks for adding this chart!

**Comment by billimek** [comment]
Oops @bjw-s I merged this one prematurely.  Instead of reverting, do you think you can make those two updates as a new PR at your leisure?

**Comment by bjw-s** [comment]
Of course, will fix them in a bit

**Pull request merged**